### PR TITLE
rgbd_to_shm server

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Merged after #16

### CODE REVIEW

File: src/viewer.cpp [hunk 1]
```diff
@@ -39,7 +38,7 @@ int main(int argc, char **argv)
                     for(int x = 0; x < view.getWidth(); ++x)
                     {
                         float d = view.getDepth(x, y);
-                        if (d == d)
+                        if (d == d) // ToDo: This will always be true.
```
Comment (reinzor) [inline src/viewer.cpp]: nan != nan I think

File: src/get_3d_point_from_image_roi_node.cpp [hunk 2]
```diff
@@ -137,12 +138,12 @@ int main(int argc, char **argv)
         last_image_stamp = ros::Time(image.getTimestamp());
 
         g_last_images_.push_back(std::shared_ptr<rgbd::Image>(new rgbd::Image(image)));
-        ROS_DEBUG("Got msg...");
+        ROS_DEBUG("[get_3d_point_from_image_roi] New image added to buffer");
```
Comment (reinzor) [inline src/get_3d_point_from_image_roi_node.cpp]: Already logged

File: src/Server_shm_only.cpp [hunk 3]
```diff
@@ -0,0 +1,156 @@
+#include "rgbd/Server_shm_only.h"
```
Comment (reinzor) [inline src/Server_shm_only.cpp]: Naming file not consistent with other classes

File: src/viewer.cpp (hunk 1 above)
Comment (MatthijsBurgh) [reply src/viewer.cpp]: Done

**Comment by MatthijsBurgh** [comment]
rgbd_to_shm  and multitool have run for more than 30 min, without any issues and stable memory usage.

**Comment by MatthijsBurgh** [comment]
@reinzor please review and approve

**Pull request merged**